pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -859,9 +867,35 @@ func (c *ClusterConfig) SetClusterStatus(cluster *eks.Cluster) error {
 	c.Status.Endpoint = *cluster.Endpoint
 	c.Status.CertificateAuthorityData = data
 	c.Status.ARN = *cluster.Arn
+
+	for k, v := range cluster.Tags {
+		if k == DisableStackPrefixTag {
+			if disableStackPrefix, err := strconv.ParseBool(*v); err == nil {
+				c.Metadata.DisableStackPrefix = disableStackPrefix
+			}
+		}
+	}
+
 	return nil
 }
 
+// MakeStackName builds a consistent name for a CloudFormation stack
+func (c *ClusterConfig) MakeStackName(suffixes ...string) string {
+	return c.MakeStackNameFromName(c.Metadata.Name, suffixes...)
+}
+
+// MakeStackNameFromName builds a consistent name for a CloudFormation stack
+func (c *ClusterConfig) MakeStackNameFromName(clusterName string, suffixes ...string) string {
+	stackName := clusterName
+	if !c.Metadata.DisableStackPrefix {
+		stackName = fmt.Sprintf("eksctl-%s", stackName)
+	}
+	for _, suffix := range suffixes {
+		stackName = fmt.Sprintf("%s-%s", stackName, suffix)
+	}
+	return stackName
+}
+
```

Comment:
Why are there two of these?

Reply:
Because of this in `manager` pkg for cluster naming: https://github.com/weaveworks/eksctl/blob/main/pkg/cfn/manager/cluster.go#L29-L42

I figured there was a reason for that second method existing there so this just became a way to leave those method signatures intact. I can go through and find what's actually using that `StackCollection.MakeStackNameFromName()` though and reorganize the code back to using just `MakeStackName()` if you think that's a better course of action.